src/test/java/io/reactivex/ParamValidationCheckerTest.java

```diff
@@ -560,6 +560,46 @@ public void checkParallelFlowable() {
 
         defaultValues.put(ParallelFailureHandling.class, ParallelFailureHandling.ERROR);
 
+        @SuppressWarnings("rawtypes")
+        class MixedConverters implements FlowableConverter, ObservableConverter, SingleConverter,
```

Comment:
Should we rename this to IdentityConverters?

Reply:
I don't feel strongly about it since this is just a stub for a test